### PR TITLE
add GetArchiveDaysMC command

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: docs/types.md [hunk 1]
```diff
@@ -283,3 +283,22 @@ Format (each line is an [extended value](#extended-value) which takes [`1..5`] b
...
+
+## Channel values by days
+
+Similarly to [channel values](#channel-values) it's a sequence of [extended values](#extended-value) to store channel pulse counter's values for days.
+<br>
+Format (each line is an [extended value](#extended-value) which takes [`1..5`] bytes):
+
+| Data                      |
+| ------------------------- |
+| channel `1` day `1` value |
```
Comment (DarkPark) [inline docs/types.md:296]: sadly this approach is not welcomed anymore

section "## Channel values with diffs" will also disappear in the next pull request

please use this approach
https://github.com/jooby-dev/jooby-docs/blob/main/docs/commands/uplink/ExAbsDayMC.md#format

File: docs/commands/GetArchiveDaysMC.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+
+## Request
+
+### Format
+
+| Size       | Type                                                         | Field                                      |
+| ---------- | ------------------------------------------------------------ | ------------------------------------------ |
+| `1`        | `byte`                                                       | command id = `0x1b`                        |
+| `1`        | `byte`                                                       | command size = `4`                         |
```
Comment (DarkPark) [inline docs/commands/GetArchiveDaysMC.md:13]:
```suggestion
| `1`        | `byte`                                                       | command size = `4+`                         |
```

?

and let's autoformat table

File: docs/commands/GetArchiveDaysMC.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+| `2`        | [packed date](../types.md#packed-date)                       | [start date](#date)                        |
+| `1..5`     | [extended value](../types.md#extended-value)                 | [channels bit set](#channels-bit-set)      |
```
Comment (DarkPark) [inline docs/commands/GetArchiveDaysMC.md:15]:
```suggestion
| `1..5`     | [extended value](../types.md#extended-value)                 | [channels bit set](../../types.md#channels-bit-set) |
```

this was approved just today and will be fixed everywhere in the next pr

File: docs/commands/GetArchiveDaysMC.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+<br>
+[See details](../../types.md#packed-date).
+
+#### **channels bit set**
+
+[See details](../../types.md#channels-bit-set).
+
+#### **days**
+
+Amount of days from archive.
```
Comment (DarkPark) [inline docs/commands/GetArchiveDaysMC.md:34]:
```suggestion
Amount of days to get data from archive.
```
?

File: docs/commands/GetArchiveDaysMC.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+| Field        | Value                     | Bits                 | Hex      |
+|--------------|---------------------------|----------------------|----------|
+| command id   | `27`                      |                      | `0x1b`   |
+| command size | `4`                       |                      | `0x04`   |
+| date         | `2023.12.23 00:00:00 GMT` | `0b0010111110010111` | `0x2f97` |
+| channels     | `1`, `3`, `4`             | `0b00001101`         | `0x0d`   |
+| days         | `2`                       |                      | `0x02`   |
+
+Message hex dump with LRC: `1b 04 2f 97 0d 02`
```
Comment (DarkPark) [inline docs/commands/GetArchiveDaysMC.md:48]: LRC is missing?

Comment (DeadByElpy) [reply docs/commands/GetArchiveDaysMC.md]: fixed

File: docs/commands/GetArchiveDaysMC.md (hunk 4 above)
Comment (DeadByElpy) [reply docs/commands/GetArchiveDaysMC.md]: fixed

File: docs/commands/GetArchiveDaysMC.md (hunk 3 above)
Comment (DeadByElpy) [reply docs/commands/GetArchiveDaysMC.md]: fixed

File: docs/commands/GetArchiveDaysMC.md (hunk 2 above)
Comment (DeadByElpy) [reply docs/commands/GetArchiveDaysMC.md]: fixed

File: docs/types.md (hunk 1 above)
Comment (DeadByElpy) [reply docs/types.md]: fixed

File: docs/commands/GetArchiveDaysMC.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+#### **channels bit set**
+
+[See details](../types.md#channels-bit-set).
+
+#### **days**
+
+Amount of days from archive.
+
+#### **channel values by days**
```
Comment (DarkPark) [inline docs/commands/GetArchiveDaysMC.md:88]: this section does not exist anymore so let's remove it

File: docs/commands/GetArchiveDaysMC.md (hunk 3 above)
Comment (DarkPark) [reply docs/commands/GetArchiveDaysMC.md]: is it commited?

Comment (DarkPark) [reply docs/commands/GetArchiveDaysMC.md]: is it commited?

Comment (DarkPark) [reply docs/commands/GetArchiveDaysMC.md]: and for Response

File: .markdownlint.json [hunk 7]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (DarkPark) [inline .markdownlint.json:1]: good idea

but maybe it's necessary to add package.json?

File: docs/commands/GetArchiveDaysMC.md (hunk 3 above)
Comment (DeadByElpy) [reply docs/commands/GetArchiveDaysMC.md]: fixed

File: .markdownlint.json (hunk 7 above)
Comment (DeadByElpy) [reply .markdownlint.json]: removed, need time to research and add good some lint tool

File: docs/commands/GetArchiveDaysMC.md (hunk 6 above)
Comment (DeadByElpy) [reply docs/commands/GetArchiveDaysMC.md]: removed

File: docs/commands/GetArchiveDaysMC.md [hunk 8]
```diff
@@ -0,0 +1,131 @@
...
+| `1`    | `byte`                                       | command size = `4+`                              |
+| `2`    | [packed date](../types.md#packed-date)       | [start date](#date)                              |
+| `1..5` | [extended value](../types.md#extended-value) | [channels bit set](../types.md#channels-bit-set) |
+| `1`    | `byte`                                       | [amount of days](#days)                          |
+
+It's a command with a [two-bytes header](../message.md#command-with-a-two-bytes-header).
+
+### Parameters
+
+#### **date**
```
Comment (DarkPark) [inline docs/commands/GetArchiveDaysMC.md:22]: it't best to match this name and the name in the Field column in Format

and for Response

Comment (DeadByElpy) [reply docs/commands/GetArchiveDaysMC.md]: fixed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**